index.html

```
@@ -1,19 +1,24 @@
-<!doctype html>
 <html lang="en">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)